api/types.proto

```diff
@@ -447,13 +447,25 @@ message AcceptancePolicy {
 }
 
 
+message ExternalCA {
+	enum CAType {
+		CFSSL = 0 [(gogoproto.enumvalue_customname) = "CATypeCFSSL"];
+	}
+
+	// CAType is the type of this external CA.
+	CAType type = 1;
+
+	// URL is the URL where the external CA can be reached.
+	string url = 2 [(gogoproto.customname) = "URL"];
```

Comment:
There's already a use-case: auth/authz information for external CA providers. I'm still looking for guidance on the ideal root CA model as right now it appears that root CA cert/key must be shared so that you auth CSR calls via previously issued certs, but generally external CAs will have their own root and authorize requests in some other fashion (in Vault's case, a token). So having a way to send this information along is a necessity.